exploration/estimate_real_user_pp/error_calc.ipynb

```diff
@@ -0,0 +1,360 @@
+    "for i in xp:\n",
+    "    xp[k] = i.timestamp()\n",
+    "    k+=1\n",
+    "\n",
+    "est_fifty_points = np.interp(date_list,list(xp),list(fp))\n",
+    "est_fifty_points"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Calculate mse for one user\n",
+    "mse = (np.square(real_fifty_points - est_fifty_points)).mean()\n",
+    "mse"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Get 10k points of real pp\n",
+    "x = []\n",
+    "real_within = real_table[real_table[:,0] >= time_comparison[first_user][\"start\"]]\n",
+    "real_within = real_within[real_within[:,0] <= time_comparison[first_user][\"end\"]]\n",
+    "real_xp = real_within[:,0]\n",
+    "real_fp = real_within[:,1]\n",
+    "\n",
+    "begin = real_xp[0] \n",
+    "end = real_xp[-1] \n",
+    "intervals = 10000 \n",
+    "real_date_list = [] \n",
+    "delta = (end - begin)/intervals \n",
+    "for i in range(1, intervals + 1): \n",
+    "    real_date_list.append((begin+i*delta).timestamp())\n",
+    "\n",
+    "k  = 0\n",
+    "for i in real_xp:\n",
+    "    real_xp[k] = i.timestamp()\n",
+    "    k+=1\n",
+    "\n",
+    "real_tenThousand_points = np.interp(real_date_list,list(real_xp),list(real_fp))\n",
+    "#real_tenThousand_points"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Get 10k points of est pp\n",
+    "x = []\n",
+    "est_within = est_table[est_table[:,0] >= time_comparison[first_user][\"start\"]]\n",
+    "est_within = est_within[est_within[:,0] <= time_comparison[first_user][\"end\"]]\n",
+    "xp = est_within[:,0]\n",
+    "fp = est_within[:,1]\n",
+    "\n",
+    "begin = xp[0] \n",
+    "end = xp[-1] \n",
+    "intervals = 10000 \n",
+    "date_list = [] \n",
+    "delta = (end - begin)/intervals \n",
+    "for i in range(1, intervals + 1): \n",
+    "    date_list.append((begin+i*delta).timestamp())\n",
+    "\n",
+    "k  = 0\n",
+    "for i in xp:\n",
+    "    xp[k] = i.timestamp()\n",
+    "    k+=1\n",
+    "\n",
+    "est_tenThousand_points = np.interp(date_list,list(xp),list(fp))\n",
+    "#est_tenThousand_points"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Get area between curve for real and curve for est\n",
+    "a = time_comparison[first_user][\"start\"].timestamp()\n",
+    "b = time_comparison[first_user][\"end\"].timestamp()\n",
+    "N = 1000 #arbitrary number\n",
+    "dx = (b - a) / N\n",
+    "x_midpoint = np.linspace(dx / 2, b - dx / 2, N)\n",
+    "real_midpoint_riemann = np.sum(real_tenThousand_points * dx)\n",
+    "est_midpoint_riemann = np.sum(est_tenThousand_points * dx)\n",
+    "area_between_curves = real_midpoint_riemann - est_midpoint_riemann\n",
+    "area_between_curves"
```

Comment:
If the curves intersect, the area difference here is inaccurate. In addition to the area, divide it by the pp range, so we get a 'density' of error/pp.

Reply:
The number also seems a little large. Provide some markdown throughout the notebook, explaining the numbers.